wandb/apis/public/api.py

```diff
@@ -1050,28 +1050,38 @@ def list_jobs(self, entity, project):
             return False
 
     @normalize_exceptions
-    def exists(self, name, type=None):
+    def artifact_exists(self, name, type=None):
```

Comment:
can you also annotate the types in the signature? I know we use the older NumPy doc style with type annotations in the docs, but I'm hoping to move away from that.

```suggestion
    def artifact_exists(self, name: str, type: Optional[str]=None) -> bool:
```